src/common/HPMDataCheck.h

```diff
@@ -321,7 +326,7 @@ HPExport const struct s_HPMDataCheck HPMDataCheck[] = {
 		#define COMMON_HPMI_H
 	#endif // COMMON_HPMI_H
 	#ifdef COMMON_MAPCHARPACKETS_H
-		{ "PACKET_MAPCHAR_AGENCY_JOIN_PARTY_REQ", sizeof(struct PACKET_MAPCHAR_AGENCY_JOIN_PARTY_REQ), SERVER_TYPE_ALL },
+		{ "PACKET_MAPCHAR_AUTH_REQ", sizeof(struct PACKET_MAPCHAR_AUTH_REQ), SERVER_TYPE_ALL },
```

Comment:
uh... is that right? I did not remove `PACKET_MAPCHAR_AGENCY_JOIN_PARTY_REQ`, but `PACKET_MAPCHAR_AUTH_REQ` comes first, are those entries just ranges?